src/pages/iou/ReceiptSelector/NavigationAwareCamera.js

```diff
@@ -1,13 +1,20 @@
+import {useTabAnimation} from '@react-navigation/material-top-tabs';
 import {useIsFocused} from '@react-navigation/native';
 import PropTypes from 'prop-types';
-import React, {useEffect, useRef} from 'react';
+import React, {useEffect, useRef, useState} from 'react';
 import {View} from 'react-native';
 import Webcam from 'react-webcam';
 
 const propTypes = {
     /* Flag to turn on/off the torch/flashlight - if available */
     torchOn: PropTypes.bool,
 
+    /* The index of the tab that contains this camera */
```

Comment:
Can you please update these comments (and the existing ones) to use the format `/** */`. This is so that storybook can automatically parse them.